lib/actionizer.rb

```diff
@@ -12,6 +12,14 @@ def self.included(base)
   end
 
   module ClassMethods
+    def to_proc
+      if new.respond_to?(:call)
+        method(:call).curry
```

Comment:
We don't have `input` here. It's only created when you go through `method_missing` with args. So whenever you say `collection.map(&my_action_class)`, you're not passing args and so there'll be no `input`.